editor/edit-post/header/mode-switcher/index.js

```diff
@@ -31,10 +31,17 @@ const MODES = [
 ];
 
 function ModeSwitcher( { onSwitch, mode } ) {
+	const filteredModes = MODES.map( MODE => {
```

Comment:
Shoud we use `mode` instead of `MODE`? It's not a module's constant.

Reply:
Right.

Reply:
mmm right, maybe `choice` or something else :)